styles/variables-silicon.less

```diff
@@ -8,6 +8,13 @@
 // to this file with the real value for the "silicon" skin.
 //
 
+// Body
+// ----------------------------------
+@agate-body-small-font-size: 25px;
+@agate-body-font-size: 28px;
+@agate-body-font-weight: normal;
+@agate-body-line-height: 1.5;
```

Comment:
Please add units to line-height (`em` in this case). This value could probably be set on some skin-global variable and reused in several places, but this is the only place right now, so we can skip that piece for now if you'd like.